### PR TITLE
filter api fix and footer alignment

### PR DESCRIPTION
**Pull request by heyrandhir**
- [x] Please merge this after [PR](https://github.com/Real-Dev-Squad/website-backend/pull/1065) and dev to main sync of backend. 
- [x] Fix the filter showing wrong results (the endpoint was not correct it was still hitting the old api not the endpoint which was created for search filters)
- [x] Position the footer at bottom when there isn't enough content. 
- [x] Position the previous and next buttons at bottom when there isn't enough content. 

*Earlier*
image

*Now*
image

### CODE REVIEW

File: users/script.js [hunk 1]
```diff
@@ -278,7 +278,9 @@ function showUserList(users) {
     imgElement.src = userData.picture?.url ?? DEFAULT_AVATAR;
     imgElement.classList.add('user-img-dimension');
     const pElement = document.createElement('p');
-    const node = document.createTextNode(`${userData.full_name}`);
+    const node = document.createTextNode(
+      `${userData.first_name} ${userData.last_name}`,
```
Comment (vikhyat187) [inline users/script.js:282]: @heyrandhir is there any reason we changed `full name` to `first + last name`?

File: users/style.css [hunk 2]
```diff
@@ -27,6 +27,27 @@
   color: var(--white-color);
 }
 
+html,
```
Comment (vikhyat187) [inline users/style.css:30]: @heyrandhir if you can share the screenshots before and after it would be really helpful.

File: users/script.js (hunk 1 above)
Comment (heyrandhir) [reply users/script.js:282]: yes earlier in filter we only had filter for status i.e OOO , idle. that status api was doing the same thing and returning from backend. the current api is returning the same as it is stored in the users collection. i.e first name and last name so I had to do it here.

File: users/style.css (hunk 2 above)
Comment (heyrandhir) [reply users/style.css:30]: sure I will add it

Comment (heyrandhir) [reply users/style.css:30]: Done @vikhyat187

**Review by vikhyat187 — APPROVED** [review]

**Review by vinit717 — APPROVED** [review]
LGTM

**Review by vikhyat187 — APPROVED** [review]

**Comment by RitikJaiswal75** [comment]
The footer is ok to bring down. But, getting the next button completely at Botton doesn't feel right

File: users/script.js [hunk 3]
```diff
@@ -271,14 +271,19 @@ async function getParticularUserData(
 
 function showUserList(users) {
   const ulElement = document.createElement('ul');
-
+  if (!users.length) {
+    userListElement.innerHTML = 'No Users Found';
+    return;
+  }
```
Comment (RitikJaiswal75) [inline users/script.js:277]: currently, this check will not work as our user API is also returning archived users. but a good thought to keep it.

**Review by RitikJaiswal75 — APPROVED** [review]

**Review by sahsisunny — APPROVED** [review]
Ship it!😜

File: users/script.js (hunk 3 above)
Comment (heyrandhir) [reply users/script.js:277]: yeah this is. for the case when the applied filter doesn't gives any user.

**Pull request merged**